stories/topics/proc-gcp-running-upgrade.adoc

```diff
@@ -2,6 +2,11 @@
 
 = Running the upgrade playbook
 
+[NOTE]
+=====
+{PlatformNameShort} upgrade to 2.4.20230630 will update the protocol of its internal load balancers. If additional networking configurations were added after installation, they may also need to be updated to ensure connectivity. See xref:tech-note-gcp-upgrade[technical note] for more detail.
```

Comment:
{PlatformNameShort} upgrade to 2.4.20230630 updates the protocol of...

they can  also require updating to ensure...

For more information, read xref:tech-note-gcp-upgrade[technical note].